onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -858,11 +865,12 @@ TEST_F(GraphTransformationTests, TransposeCastMatmulFusion) {
     graph_transformation_mgr.Register(onnxruntime::make_unique<MatmulTransposeFusion>(), TransformerLevel::Level1);
     auto ret = graph_transformation_mgr.ApplyTransformers(graph, TransformerLevel::Level1, *logger_);
     ASSERT_TRUE(ret.IsOK());
+    Model::Save(*p_model, "model_" + std::to_string(i++) + ".onnx");
     std::map<std::string, int> op_to_count = CountOpsInGraph(graph);
     ASSERT_TRUE(op_to_count["Transpose"] == 0);
     ASSERT_TRUE(op_to_count["MatMul"] == 0);
     ASSERT_TRUE(op_to_count["Cast"] == 2);
-    ASSERT_TRUE(op_to_count["com.microsoft.FusedMatMul"] == 1);
+    ASSERT_TRUE(op_to_count["com.microsoft.FusedMatMul"] >= 1);
```

Comment:
>>= [](start = 57, length = 2)

does it depend on i ? if so, u can check == 1 when i == something. and ==2 when i is another

Reply:
I want to make it independent of the index (i) so that if the test cases are moved up and down. All scenarios require the count >=1.

Reply:
I think this is a good idea in general but in this unittest case not necessary.